packages/amplication-nest-logger-module/src/index.ts

```diff
@@ -1,3 +1,9 @@
+import winston from "winston/lib/winston/config";
+
 export * from "./logger.module";
 export { WINSTON_MODULE_PROVIDER as AMPLICATION_LOGGER_PROVIDER } from "nest-winston";
-export { Logger as AmplicationLogger } from "winston";
+export {
+  Logger as AmplicationLogger: Logger,
+  transports as Transports,
+  createLogger as CreateLogger,
+} from "winston";
```

Comment:
The reason is because of the below question:
>There are places were winston is [directly used](https://github.com/amplication/amplication/pull/3553/files#diff-0bd5f3f111abe353c918c42195a3ecfc287ced63e6debf18d4d0ddda5a49297aL391) in the build.service.ts, I could not find a way to remove/replace the usage. Should these be retained as it is?

By the above question I meant that there were instances in `build.service` which were using `transports` and `createLogger` from `winston`.
I had to either keep it as it is or move it to `amplication-nest-logger-module`. I thought of moving as there could be similar references in other project. Let me know if I should rever these changes?